### DISCUSSION TITLE
Install @react-native-firebase/crashlytics v6 with react-native-firebase v5

### DISCUSSION THREAD

**Discussion by meliodev** [invertase/react-native-firebase#4740] [Q&A] [answered]
Hello, 
I am using react-native-firebase v5. For the moment, I can't migrate to v6 due to deadline and cost issues. 
Now I am trying to make @react-native-firebase/crashlytics v6 work with react-native-firebase v5 modules but I get conflicts. 
How can I disable/uninstall Crashlytics of react-native-firebase v5 to allow @react-native-firebase/crashlytics work properly ? Or is there any solution to realize that ?
Cheers

**Comment by mikehardy** [comment, marked as answer]
There is zero support for v5, no time will be spent on it here, sorry. You'll have to petition your management in stronger terms. If you check the issues list you will find one or two adventurous folks who have hack-and-slashed modern `@react-native-firebase/crashlytics` into an otherwise `react-native-firebase@v5` solution but I can only discourage it in the strongest terms.
